### PR TITLE
refactor: Get asset from Output Module

### PR DESCRIPTION
**Pull request by aronhoyer**
Resolves [AB#106056](https://dev.azure.com/EquinorASA/d02c350d-092d-41bf-a812-f05d5d8ad1b0/_workitems/edit/106056). Depends on #57

### CODE REVIEW

File: GetAssetTrigger/index.ts [hunk 1]
```diff
@@ -37,34 +37,51 @@ import { toNetError } from "../lib/net/to_net_err"
...
-    TE.map(({ approvedAsset, collibraAttributes, collibraCommunity, tags }) =>
-      assetAdapter(collibraAttributes)(collibraCommunity)(tags)(approvedAsset)
+    context.bindingData.id,
+    isValidID,
+    E.mapLeft(toNetError(400)),
+    TE.fromEither,
+    TE.chain(getAsset(collibraClient)),
+    TE.mapLeft(toNetError(500)),
+    TE.map(
+      (asset) =>
```
Comment (wenche) [inline GetAssetTrigger/index.ts:51]: The map function might be a separate function to make this code a little bit shorter and more readable

File: GetAssetsTrigger/index.ts [hunk 2]
```diff
@@ -38,37 +34,46 @@ import { makeResult } from "../lib/net/make_result"
...
-            TE.map(({ attributes, community, tags }) => assetAdapter(attributes)(community)(tags)(asset))
-          )
-        ),
-        TE.sequenceArray
+    TE.of(collibraClient),
+    TE.bind("assets", getAssets),
+    TE.mapLeft(toNetError(500)),
+    TE.map(({ assets }) =>
+      assets.flatMap(
+        (asset) =>
```
Comment (wenche) [inline GetAssetsTrigger/index.ts:47]: Same, might clean things up a bit if it was a separate function 🤔

**Review by wenche — APPROVED** [review]
LGTM 👍  

But suggested a separate map function

**Comment by aronhoyer** [comment]
> But suggested a separate map function

yeah, that's gonna be part of a later pr

**Pull request merged**